pkg/server/image_pull.go

```diff
@@ -347,9 +345,16 @@ func (c *criService) registryHosts(auth *runtime.AuthConfig) docker.RegistryHost
 	}
 }
 
-// addDefaultEndpoint add default registry endpoint if it does not
-// exist in the passed-in endpoint list.
-func addDefaultEndpoint(endpoints []string, host string) ([]string, error) {
+// registryEndpoints returns  endpoints for a given host. It adds default registry endpoint if it
```

Comment:
s/  / /

Reply:
should mention wildcarding.. here and in the md... to explain that host configured endpoints will be used first for matching host operations, then, if specified, wildcard specified host endpoints will be used, then if the default registry endpoint is not already specified via hostname or wildcard it will be added to the endpoints list for the given host.

Reply:
Done.